backend/src/test/resources/fhir/observationGenderOfSexualPartners.json

```diff
@@ -0,0 +1,154 @@
+[
+  {
+    "resourceType": "Observation",
+    "id": "8526bd3b-42e4-3a6f-88ff-3fc43b69dc20",
+    "identifier": [
+      {
+        "use": "official",
+        "type": {
+          "coding": [
+            {
+              "system": "http://loinc.org",
+              "code": "81959-9",
+              "display": "Public health laboratory ask at order entry panel"
+            }
+          ]
+        }
+      }
+    ],
+    "status": "final",
+    "code": {
+      "coding": [
+        {
+          "system": "http://simplereport.gov/",
+          "code": "SR0001",
+          "display": "What is the gender of their sexual partners"
+        }
+      ],
+      "text": "What is the gender of their sexual partners"
+    },
+    "valueCodeableConcept": {
+      "coding": [
+        {
+          "system": "http://snomed.info/sct",
+          "code": "446131000124102",
+          "display": "non-binary"
```

Comment:
Based on what the ticket says it looks like we want non binary to be displayed as the following:
```
Nonbinary or gender non-conforming => system: http://snomed.info/sct, code: 446131000124102, display: Non-binary gender identity
```

however the [`genderIdentityDisplaySet`](https://github.com/CDCgov/prime-simplereport/blob/c47bc17e95f7365ebaf1e282532434b226274cf4/backend/src/main/java/gov/cdc/usds/simplereport/db/model/PersonUtils.java#L827) has `nonbinary` set to `"non-binary"` is this something we can change?

Reply:
Cleaned this up in both places!